.github/workflows/test_firedrake_install.yml

```
@@ -0,0 +1,89 @@
+name: 'Test Firedrake Installation'
+
+on:
+  # Trigger when this or any file in the install directory changes
+  pull_request:
+    paths:
+      - 'install/**'
+      - '.github/workflows/test_firedrake_install.yml'
+
+  # Trigger the workflow on the 1st day of every month at midnight
+  schedule:
+    - cron: '0 0 1 * *'
+
+  # Manually trigger the workflow
+  workflow_dispatch:
+
+# Stop the workflow if a new run is requested
+concurrency:
+  group: 'install_firedrake_test-${{ github.ref }}'
```

```suggestion
  group: 'test_firedrake_install-${{ github.ref }}'
```
Forgot to rename this to match filename, sorry